.evergreen/releases.yml

```diff
@@ -0,0 +1,55 @@
+exec_timeout_secs: 3600 
+
+functions:
+  "fetch source":
+    - command: git.get_project
+      type: system
+      params:
+        directory: "src"
+
+  "install dependencies":
+    command: shell.exec
+    params:
+      working_dir: "src"
+      script: |
+        ${PREPARE_SHELL}
+        .evergreen/install-dependencies.sh
+
+  "publish release":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: "src"
+        script: |
+          set +x
+
+          TAG=${GIT_TAG}           \
+          TOKEN=${CRATES_IO_TOKEN} \
```

Comment:
is this token set via the evg UI / project settings?

Reply:
Yep. Note that `set +x` means "disable tracing", not enable, so this won't get logged